recipes/libsmacker/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from contextlib import contextmanager
+import os
+
+
+class LibsmackerConan(ConanFile):
+    name = "libsmacker"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://libsmacker.sourceforge.net"
+    topics = ("conan", "libsmacker", "decoding ", "smk", "smacker", "video", "file")
+    license = "LGPL-v2.1"
```

Comment:
```suggestion
    license = "LGPL-2.1-or-later"
```

https://spdx.org/licenses/LGPL-2.1-or-later.html